src/mermaid.tsx

```diff
@@ -18,6 +18,15 @@ export default function (props: { content: string }) {
       .then(() => {
         timer = setTimeout(() => {
           setReady(true)
+
+          let theme = "default"
+          if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
```

Comment:
This detects the system's prefered theme. It might be better to use `preferredThemeMode` in `logseq.App.getUserConfigs()`